src/spyglass/common/common_ephys.py

```diff
@@ -114,6 +115,9 @@ def make(self, key):
         else:
             electrode_config_dicts = dict()
 
+        # properties that are optional when creating NWB file but not part of trodes_to_nwb
+        optional_fields = ["x", "y", "z", "filtering", "impedance"]
+
```

Comment:
If using `get` or `getattr`, no need for this 
```suggestion
```

Reply:
Removed this